dodola/tests/test_services.py

```diff
@@ -197,27 +206,22 @@ def test_regrid_methods(regrid_method, expected_shape):
         out="an/output/path.zarr",
         method=regrid_method,
         storage=fakestorage,
+        domain_file="a/domainfile/path.zarr",
     )
     actual_shape = fakestorage.read("an/output/path.zarr")["fakevariable"].shape
     assert actual_shape == expected_shape
 
 
 @pytest.mark.parametrize(
-    "target_resolution, expected_shape",
+    "expected_shape",
     [
         pytest.param(
-            1.0,
             (180, 360),
-            id="Regrid to global 1.0° x 1.0° grid",
-        ),
-        pytest.param(
-            2.0,
-            (90, 180),
-            id="Regrid to global 2.0° x 2.0° grid",
+            id="Regrid to domain file grid",
```

Comment:
I'm afraid that these changes in this `@pytest.mark.parametrize` for `test_regrid_resolution` loose the purpose of the test.

(granted I was sloppy and the docstr did not document the purpose of the test very well...)

Maybe we could keep the behavior this test is checking for -- mainly, that different domain file resolutions should create a corresponding change in regridded output resolution.

I think one easy way to get this is adding back that `target_resolution` arg in the parameterize and then passing it to the `global_grid()` call below.

If you don't think it's important to check this behavior anymore then remove the test all together.

Reply:
I made some fancy updates - I _think_ it works now.

Reply:
Innnnteresting! Looks good to me, and if it doesn't work - we can fix it. Nice work. Marking resolved.